src/views/LeadAdd/index.js

```diff
@@ -397,7 +397,7 @@ class LeadAdd extends React.PureComponent {
                     publishedOn: formatDateToString(now, 'yyyy-MM-dd'),
                     // NOTE: hard-coded confidentiality value
                     confidentiality: 'unprotected',
-                    // Inject other values too
+                    authors: isDefined(faramValues.authors) ? faramValues.authors : [],
```

Comment:
Why was authors added here? And why not other values like publishers?

Reply:
The field is optional, when passed null, it throws an error